SU-SWS/stanford_profile_helper
# D8CORE-3053: fixing margin bottom on lists as cards

# READY FOR REVIEW

# Summary
- Adjusting the margin bottom for the card list view for news and events when in a column.

# Needed By (Date)
- 1/26/21

# Urgency
- Low

# Steps to Test

1. Pull in this change.
2. make card lists of Events, News and people in one row. This makes it easier to see the margin size difference.

# Affected Projects or Products
- List paragraphs on SBT
- List paragraph on SOE

# Associated Issues and/or People
- D8CORE-3053
- SOE 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
